src/dialogs.ts

```diff
@@ -50,21 +54,44 @@ export const nativeMessage = ({
     duration,
     buttonText,
     type,
+    withDismiss,
 }: {
     message: string;
-    duration?: number; // milliseconds
+    /**
+     * By default, the snackbar auto dismisses after 5s or 10s (if it has buttonText)
+     * PERSISTENT: the snackbar won't dismiss without user interaction
+     * FIVE_SECONDS: the snackbar will dismiss after 5 seconds
+     * TEN_SECONDS: the snackbar will dismiss after 10 seconds
+     * number: deprecated, this is ignored by native app
```

Comment:
I think we should add which default values are used if a numeric duration is passed to the method:
If `buttonText` is `null` then `FIVE_SECONDS`, else `TEN_SECONDS`

And maybe is it worthy to mention that:
- `FIVE_SECONDS` will be set instead when the method receives no `buttonText` and `TEN_SECONDS` as its duration
- `TEN_SECONDS` will be set instead when the method recieves a non-null `buttonText` and `FIVE_SECONDS` as its duration

Reply:
hmmm... I have a doubt with this, then why do we need those enum values? I mean, when can I use `TEN_SECONDS` or `FIVE_SECONDS`?, it seems like those values will be always ignored, wont't they?

Reply:
removed `TEN_SECONDS` and `FIVE_SECONDS` from the enum